gridcapa-core-cc-app/src/main/resources/application.yml

```
@@ -1,0 +1,4 @@
+spring:
+  cloud:
+    function:
+      definition: request
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)